src/postgres/include/c.h

```diff
@@ -289,17 +289,6 @@ typedef unsigned long long int uint64;
 #define UINT64_FORMAT "%" INT64_MODIFIER "u"
 
 /*
- * 128-bit signed and unsigned integers
- *		There currently is only a limited support for the type. E.g. 128bit
- *		literals and snprintf are not supported; but math is.
- */
-#if defined(PG_INT128_TYPE)
-#define HAVE_INT128
-typedef PG_INT128_TYPE int128;
-typedef unsigned PG_INT128_TYPE uint128;
-#endif
-
-/*
```

Comment:
Do we actually need to remove this, given PG_INT128_TYPE is not defined now? (would be good if you can test, since its easier if the change is just in pg_config.h due to code generation)